src/compiler/cicc.rs

```diff
@@ -200,17 +197,16 @@ where
     }
 
     if let Some(module_id_path) = module_id_file_name {
-        if gen_module_id_file {
-            outputs.insert(
-                "--module_id_file_name",
-                ArtifactDescriptor {
-                    path: module_id_path,
-                    optional: true,
-                },
-            );
-        } else {
-            extra_dist_files.push(module_id_path);
+        if module_id_path.exists() {
```

Comment:
If the generated file doesn't exist, does this exhibit an issue or is it ok to silently ignore?